src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -846,136 +846,6 @@ public static function xpathLiteral($s)
     }
 
     /**
-     * @deprecated Using the SplObjectStorage API on the Crawler is deprecated as of 2.8 and will be removed in 3.0.
-     */
-    public function attach($object, $data = null)
```

Comment:
removing this without actually removing `extends \SplObjectStorage` makes no sense